src/Raven.Server/Documents/Indexes/MapIndexBase.cs

```diff
@@ -41,8 +41,11 @@ public override void HandleDelete(DocumentTombstone tombstone, string collection
 
         public override int HandleMap(LazyStringValue key, IEnumerable mapResults, IndexWriteOperation writer, TransactionOperationContext indexContext, IndexingStatsScope stats)
         {
-            if (_filter.Add(key) == false)
-                writer.Delete(key, stats);
+            using (var bloomStats = stats.For(IndexingOperation.Map.Bloom))
+            {
+                if (_filter.Add(key) == false)
+                    writer.Delete(key, bloomStats);
```

Comment:
We are charging the delete to the bloom as well?
It is just registering the delete here (so would be quick), or actually doing that?